apps/hash-graph/bin/cli/src/subcommand/test_server.rs

```diff
@@ -20,13 +27,23 @@ pub struct TestServerArgs {
 
     /// The address the REST client is listening at.
```

Comment:
Perhaps there is another instance of this comment for the main graph server?
```suggestion
    /// The address the REST server is listening at.
```